### ISSUE TITLE
problems with training

### ISSUE THREAD

**Issue by Mirorrn** [experiencor/basic-yolo-keras#3]
Hallo,
i have a problem to get the same training results as you with the last layer reset. The SGD optimizer didnt work for me i only got results with adam.
Like in darkflow i dont understand why the mean of the loss function is mult with .5
"loss = .5 * tf.reduce_mean(loss)" i didn´t saw it in the yolo-paper.

**Comment by experiencor** [comment]
I have to admit that the training doesn't converge all the times. But my experience is that I have more luck with SGD than Adam. Adam usually leads to NAN loss half-way. Regarding SGD, if it's converging then it will never result in NAN loss.

0.5 is just a scale factor. It means nothing at all. Just for consistency purpose as we usually add 1/2 to mean square difference to simplify the math.
